addOns/websocket/src/main/java/org/zaproxy/zap/extension/websocket/WebSocketProxyV13.java

```diff
@@ -717,10 +716,9 @@ public String getReadablePayload() {
             } catch (InvalidUtf8Exception e) {
                 if (logger.isDebugEnabled()) {
```

Comment:
Remove

Reply:
I left them on purpose, to avoid creating the strings always.